packages/compass-components/src/components/signal-popover.tsx

```diff
@@ -115,10 +115,13 @@ const SignalCard: React.FunctionComponent<
       data-signal-id={id}
     >
       <strong className={signalCardTitleStyles}>{title}</strong>
-      <div className={signalCardDescriptionStyles}>{description}</div>
+      <Body as="div" baseFontSize={13} className={signalCardDescriptionStyles}>
```

Comment:
Should the font size be a constant somewhere?

Reply:
Might be able to use `BaseFontSize` from https://github.com/mongodb/leafygreen-ui/blob/72999c2cf97d03201cb6e6b0fe0df2233f0effcc/packages/tokens/src/index.ts#L9

Reply:
This is a [enum value from BaseFontSize](https://github.com/mongodb/leafygreen-ui/blob/72999c2cf97d03201cb6e6b0fe0df2233f0effcc/packages/tokens/src/typeScales.ts#L2), you can't really pass anything else here